test/lib/fixtures.ts

```diff
@@ -64,181 +74,51 @@ export interface ArbitrumL1Mocks {
-    await controller.setContractProxy(utils.id('RewardsManager'), rewardsManager.address)
-    await controller.setContractProxy(utils.id('ServiceRegistry'), serviceRegistry.address)
-    await controller.setContractProxy(utils.id('GNS'), gns.address)
-    if (isL2) {
-      await controller.setContractProxy(utils.id('GraphTokenGateway'), l2GraphTokenGateway.address)
-    } else {
-      await controller.setContractProxy(utils.id('GraphTokenGateway'), l1GraphTokenGateway.address)
-    }
-
-    // Setup contracts
-    await curation.connect(deployer).syncAllContracts()
-    await gns.connect(deployer).syncAllContracts()
-    await serviceRegistry.connect(deployer).syncAllContracts()
-    await disputeManager.connect(deployer).syncAllContracts()
-    await rewardsManager.connect(deployer).syncAllContracts()
-    await staking.connect(deployer).syncAllContracts()
-    if (isL2) {
-      await l2GraphTokenGateway.connect(deployer).syncAllContracts()
-    } else {
-      await l1GraphTokenGateway.connect(deployer).syncAllContracts()
-      await bridgeEscrow.connect(deployer).syncAllContracts()
-      await grt.connect(deployer).addMinter(l1GraphTokenGateway.address)
-    }
-
-    await staking.connect(deployer).setSlasher(slasherAddress, true)
-    await gns.connect(deployer).approveAll()
-    await grt.connect(deployer).addMinter(rewardsManager.address)
-
-    // Unpause the protocol
-    await controller.connect(deployer).setPaused(false)
-
-    if (isL2) {
-      return {
-        controller,
-        disputeManager,
-        epochManager,
-        grt: grt as L2GraphToken,
-        curation,
-        gns,
-        staking,
-        libExponential,
-        rewardsManager,
-        serviceRegistry,
-        proxyAdmin,
-        l2GraphTokenGateway,
-      } as L2FixtureContracts
-    } else {
-      return {
-        controller,
-        disputeManager,
-        epochManager,
-        grt: grt as GraphToken,
-        curation,
-        gns,
-        staking,
-        libExponential,
-        rewardsManager,
-        serviceRegistry,
-        proxyAdmin,
-        l1GraphTokenGateway,
-        bridgeEscrow,
-      } as L1FixtureContracts
-    }
-  }
-
-  async load(
-    deployer: Signer,
-    slasher: Signer = Wallet.createRandom() as Signer,
-    arbitrator: Signer = Wallet.createRandom() as Signer,
-  ): Promise<L1FixtureContracts> {
-    return this._loadLayer(deployer, slasher, arbitrator, false) as unknown as L1FixtureContracts
-  }
+    // Post deploy configuration
+    await contracts.GraphToken.connect(deployer).addMinter(deployer.address)
+    await contracts.Controller.connect(deployer).setPaused(false)
 
-  async loadL2(
-    deployer: Signer,
-    slasher: Signer = Wallet.createRandom() as Signer,
-    arbitrator: Signer = Wallet.createRandom() as Signer,
-  ): Promise<L2FixtureContracts> {
-    return this._loadLayer(deployer, slasher, arbitrator, true) as unknown as L2FixtureContracts
+    return contracts
```

Comment:
As mentioned in our call, returning `contracts as L1FixtureContracts` or `as L2FixturContracts` would solve a lot of the type wrangling in the tests, but can be addressed in a separate PR

Reply:
Definitely a better approach. Though the ultimate goal is to get rid of `L1FixtureContracts` and `L2FixtureContracts` in favor of `GraphNetworkContracts`. 

Created #885